notebooks/lectures/Introduction_to_Pairs_Trading/notebook.ipynb

```diff
@@ -978,7 +978,7 @@
    "source": [
```

Comment:
👍 on the fix to the source. It looks like the cell output is also stored in the notebook. Should we re-run the cell and save the new output?